webserver/pkgpkr/webservice/templates/webservice/about.html

```diff
@@ -17,13 +40,13 @@ <h3>How do we calculate the Package Picker Score?</h3>
 
               </div>
               <div class="about-sub-div">
-                  <img src="/static/images/happiness.jpg" />
+                  <img src="/static/images/how-it-works/overall.jpg" />
```

Comment:
because the file is overall.png, it does not display the image, but it is easy to fix so nothing too worry about